ethportal-peertest/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "ethportal-peertest"
```

Comment:
nitpick - I'm not sure I love this name - but this is a minor subjective nitpick. I'm assuming with the `ethportal` prefix - you're implying that it's an implementation agnostic testing framework? And the `peertest` suffix is somewhat confusing for me. Some ideas off the top of my head (which tbh I'm not sure are any better) `portal-test-harness` /  `test-framework` / `test-harness` / `testing-node` ... again, totally subjective nitpick here, not a blocker for me at all.

Reply:
The crate name was a product of [this](https://github.com/ethereum/trin/pull/79#discussion_r694191099) conversation. Happy to change it any time.